R/reexports.R

```diff
@@ -0,0 +1,8 @@
+#' @export
+dplyr::vars
+
+#' @export
+dplyr::desc
+
```

Comment:
We should also reexport `rlang::exprs()` which is required for calling `derive_var_basetype()`.

Reply:
I'm running into this issue _but _only_ for the R 4.0 check, any advice?
![image](https://user-images.githubusercontent.com/43018762/204367889-fdb79773-0326-4ffa-875d-48f09b2e9041.png)